src/main/java/be/seeseemelk/mockbukkit/entity/EntityMock.java

```diff
@@ -61,9 +61,11 @@
 
 public abstract class EntityMock extends Entity.Spigot implements Entity, MessageTarget
 {
+	private static int idIndex = 0;
 
 	private final @NotNull ServerMock server;
 	private final @NotNull UUID uuid;
+	private final int entityId;
```

Comment:
Should be renamed to `id`. It's already in the entity class so specifying entity is redundant.